lib/hanami/model/plugins/timestamps.rb

```diff
@@ -24,7 +24,7 @@ class InputWithTimestamp < WrappingInput
           def initialize(relation, input)
             super
             columns     = relation.columns.sort
```

Comment:
What's the point of sorting the columns here, or even instantiating `columns`? :confused: